lisa/sut_orchestrator/azure/README.rst

```diff
@@ -0,0 +1,36 @@
+Azure Deployment Template
+==========================
+
+`Bicep <https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/>`__ provides a more concise and readable way to define Azure infrastructure compared to ARM templates. However, as of now, the Azure Python SDK doesn't support direct deployment of Bicep templates. Therefore, we need to generate an ARM template from the Bicep template and deploy it using the Azure SDK.
```

Comment:
To fast respond to regression on the big change, please add a flag in dev section, called `use_bicep`. The default value is `false`. And make the generated json another name like `autogen_arm_template.json`.

Reply:
you mean add use_bicep into runbook, by default, set it as false?

Reply:
Yes, add a use_bicep. The default value depends on how we confident on the change. If no regression on wide scenarios, the default value is true, otherwise set to false. If it's set to true, and a regression is reported in some pipelines, the value could be set to false there. It doesn't need to wait we fix the regression.

Reply:
updated